README.md

```diff
@@ -1,4 +1,15 @@
-**Points:** all: 0 target: 200
+Player      | order | points
+----------- | ----- | ------
+akvadrako   | Ba    | 10
+RamonRB     | Be    | 0
+rknegjens   | K     | 0
```

Comment:
I propose to change this to `rknegjens   | K     | 100`

Reply:
I was thinking it's a good way to continue playing if many people drop out, but I guess it is unsafe.